src/Output/ProgressBar.php

```diff
@@ -166,7 +166,9 @@ public function each($items, callable $callback = null)
             $items = iterator_to_array($items);
         }
 
-        $total = count($items);
+        if (0 === $total = count($items)) {
```

Comment:
ooh oops, i think i didn't know how to format code suggestion of github.
the idea was to combine assignment and checks in oneliner
```php
$total = count($items) 
if (!$total) {
    return;
}

if (0 === $total = count($items)) {
    return;
}
```